### DISCUSSION TITLE
run the tests serially between assemblies

### DISCUSSION THREAD

**Discussion by yoda07** [xunit/xunit#2284] [Question] [answered]
I am trying to make my tests run serially between assemblies. I have read the documents and try using the configuration json file with `"parallelizeAssembly": false`. But it doesn't work nether in dotnet test nor visual studio. I even tested it with the official examples:
assembly 1:
```
namespace TestProject1
{
    public class TestClass1
    {

        [Fact]
        public void Test1()
        {
            Thread.Sleep(3000);
        }
    }
}
```
assembly 2:
```
namespace TestProject2
{
    public class TestClass2
    {

        [Fact]
        public void Test2()
        {
            Thread.Sleep(3000);
        }
    }
}

```
And the total complete time is around 4 seconds.
Where did I do wrong?

**Comment by bradwilson** [comment, marked as answer]
Unfortunately, this is because `dotnet test` and Visual Studio (both of which are using the VSTest runner system) don't obey our settings. This is really a question of how to disable parallelism with VSTest and not with xUnit.net, unfortunately, so you're probably better off asking the question over there: https://github.com/microsoft/vstest
